index.d.ts

```diff
@@ -17,7 +18,7 @@ interface FocusTrapProps extends React.HTMLProps<FocusTrap> {
   component?: React.Component | string;
 }
 
-interface HotKeysProps extends FocusTrapProps<HotKeys> {
+interface HotKeysProps extends React.Props<HotKeys>, FocusTrapProps {
```

Comment:
Was the problem that `HotKeyProps` wasn't correctly inheriting `React.HTMLProps` through `FocusTrapProps`, or that `React.HTMLProps` is not correct (doesn't contain `className`), but `React.Props` is?

Reply:
I would have to tinker about again to figure out exactly what the deal is with `className`. From memory it was happy once I added the second import, but I could be misremembering.

Somewhere along the lines you'll need `React.Props` to get the three in-built props, one of which I needed in my project

Reply:
Can we try using `React.HTMLAttributes`? It seems [React.Props has been deprecated](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/8509#issuecomment-197073806)

Reply:
Using `React.HTMLAttributes`. I had to create definitions for `onFocus` and `onBlur` since your use of them seems to override React's own